src/app/ApplicationTemplate.Tests/Framework/IntegrationTestBase.cs

```diff
@@ -27,6 +30,10 @@ public IntegrationTestBase()
 			ConfigurationSetup(ConfigureHost);
 		}
 
+		/// <summary>
+		/// Initializes different services used by the app.
+		/// </summary>
+		/// <param name="extraHostConfiguration">Add specific configurations for project initialization.</param>
 		protected void ConfigurationSetup(Action<IHostBuilder> extraHostConfiguration = null)
```

Comment:
ConfigurationSetup is a weird method name. Methods should start with a verb. Looking at the summary, it should be "InitializeApplication" or "InitializeService".

Reply:
The method `InitializeAsync` should probably be renamed to "StartApplication" or "StartServices".

Reply:
InitializeAsync comes from the IAsyncLifetime interface thought. I cannot do anything about that